README.md

```diff
@@ -30,24 +34,42 @@ This team is spun off from the [Modules team](https://github.com/nodejs/modules)
 
 - [x] Implement chaining as described in the [design](doc/design/proposal-chaining-middleware.md), where the `default<hookName>` becomes `next` and references the next registered hook in the chain. https://github.com/nodejs/node/pull/42623
 
-- [ ] Move loaders off thread
+- [ ] Move loaders off thread.
 
-- [ ] Add helper/utility functions [`module`](https://nodejs.org/api/module.html) module
+   We hope that moving loaders off thread will allow us to preserve an async `resolve` hook while supporting the sync `import.meta.resolve` API. If that turns out to be unachievable, however, then:
 
-   - [ ] Start with the functions that make up the ESM resolution algorithm as defined in the [spec](https://nodejs.org/api/esm.html#resolver-algorithm-specification). Create helper functions for each of the functions defined in that psuedocode: `esmResolve`, `packageImportsResolve`, `packageResolve`, `esmFileFormat`, `packageSelfResolve`, `readPackageJson`, `packageExportsResolve`, `lookupPackageScope`, `packageTargetResolve`, `packageImportsExportsResolve`, `patternKeyCompare`. (Not necessarily all with these exact names, but corresponding to these functions from the spec.)
-   
-   - [ ] Follow up with similar helper functions that make up what happens within Node’s internal `load`. (Definitions to come.)
+   - [ ] Convert `resolve` from async to sync https://github.com/nodejs/node/pull/43363
+
+      - [ ] Add an async `resolve` to [`module`](https://nodejs.org/api/module.html) module
+
+      - [ ] Consider an API for async operations before resolution begins, such as `preImport` https://github.com/nodejs/loaders/pull/89
+
+- [ ] Provide a way to register loaders without a command-line flag, for example via a `"loaders"` field in `package.json`.
 
 - [ ] Support loading source when the return value of `load` has `format: 'commonjs'`. See https://github.com/nodejs/node/issues/34753#issuecomment-735921348 and https://github.com/nodejs/loaders-test/blob/835506a638c6002c1b2d42ab7137db3e7eda53fa/coffeescript-loader/loader.js#L45-L50.
 
+- [ ] Finish and stabilize `--experimental-vm-modules` so that `vm` fully supports ESM.
```

Comment:
While I agree this is something Node.js should make progress on and it would be awesome to see some progress on it, I'm not sure this is related to loaders – it seems to me that the work that needs to happen on loaders and the work that needs happen on `node:vm` have very little overlap and don't face the same challenges (but maybe I'm wrong on that). If there's little overlap, I don't think it makes sense to keep it in "Milestone 1", instead it would need to go on a "parallel thread".

In any case, let's use the `node:` prefix when talking about core modules so there's no ambiguity.
```suggestion
- [ ] Finish and stabilize `--experimental-vm-modules` so that `node:vm` fully supports ESM.
```